app/components/Account/Proposals.jsx

```diff
@@ -252,6 +266,8 @@ class Proposals extends Component {
             const id = proposal.proposal.get("id");
             const proposer = proposal.proposal.get("proposer");
             const expiration = proposal.proposal.get("expiration_time");
+            const operation = proposal.operations && proposal.operations.size &&
```

Comment:
Why restrict this to only the first operation? What is the intention?

Reply:
I didn't know that we can create proposal with a few operations.
fixed.